ci/docker/amdgpu-release-rocm523.yaml

```diff
@@ -10,7 +10,7 @@
 
 spack:
   specs:
-  - dla-future@develop +rocm +miniapps +ci-test amdgpu_target=gfx906:xnack- ^openblas ^mpich
+  - dla-future@develop +rocm +miniapps +ci-test amdgpu_target=gfx906:xnack- ^openblas ^mpich ^ncurses@6.2
```

Comment:
Could you revert this and then do a bors try? Or am I missing the reason for this change?